src/main/scala/nl/knaw/dans/bag/DansBag.scala

```diff
@@ -384,6 +384,39 @@ trait DansBag {
    *         `scala.util.Failure` otherwise
    */
   def save(): Try[Unit]
+
+  /**
+   * Verifies if a bag is complete.
+   * According to the BagIt v16 specs, a bag is complete when:
```

Comment:
use spelled out "version 16" for the release "versions" and "V1.0", "V0.97" for main versions.